src/config.py

```diff
@@ -0,0 +1,31 @@
+from configparser import ConfigParser
+
+CONFIG_FILE_NAME = "config.ini"
+DEFAULT_SECTION_NAME = "GitHub"
+
+
+class GithubConfig:
+    config = ConfigParser()
+
+    def setup_config(self: "GithubConfig") -> None:
+        self.config.read(CONFIG_FILE_NAME)
+        if DEFAULT_SECTION_NAME not in self.config.sections():
+            self.config.add_section(DEFAULT_SECTION_NAME)
+
+    def get(
+        self: "GithubConfig",
+        key: str,
+        fallback: str = "",
+        section: str = DEFAULT_SECTION_NAME,
+    ) -> str:
+        return self.config.get(section, key, fallback=fallback)
+
+    def update(
+        self: "GithubConfig",
```

Comment:
**suggestion:** python 3.11 has a [special shortcut type](https://docs.python.org/3/library/typing.html?highlight=typing%20self#typing.Self) for the explicit purpose of typing self
```suggestion
from typing import Self

from configparser import ConfigParser

CONFIG_FILE_NAME = "config.ini"
DEFAULT_SECTION_NAME = "GitHub"


class GithubConfig:
    config = ConfigParser()

    def setup_config(self: Self) -> None:
        self.config.read(CONFIG_FILE_NAME)
        if DEFAULT_SECTION_NAME not in self.config.sections():
            self.config.add_section(DEFAULT_SECTION_NAME)

    def get(
        self: Self,
        key: str,
        fallback: str = "",
        section: str = DEFAULT_SECTION_NAME,
    ) -> str:
        return self.config.get(section, key, fallback=fallback)

    def update(
        self: Self,
```

Reply:
Great to know 🌟